.env.example

```
@@ -4,7 +4,7 @@
 # If you need an app_code for local testing, we'll give you one for free:
 # opensource@hichroma.com
 
-CHROMATIC_APP_CODE=<place app_code here>
+CHROMATIC_PROJECT_CODE=<place app_code here>
```

```suggestion
CHROMATIC_PROJECT_TOKEN=<place app_code here>
```

(And a bunch of other places below)